toolchain/mfc/params_tests/coverage.py

```
@@ -0,0 +1,278 @@
+    for node in ast.walk(tree):
+        if isinstance(node, ast.FunctionDef):
+            current_method = node.name
+            continue
+
+        if not _is_prohibit_call(node):
+            continue
+
+        message = _extract_message(node.args[1])
+        try:
+            condition_code = ast.unparse(node.args[0])
+        except Exception:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)